components/custom/Chat.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Badge } from '../ui/badge';
+import { twMerge } from 'tailwind-merge';
+
+interface Props {
+  text: string;
+  className?: string;
+}
+
+export function SpeechBubble({ className, text }: Props) {
+  return (
+    <Badge
+      className={twMerge(
+        `inline bg-violet-700 text-white hover:bg-violet-500 text-[16px] w-[140px] ${
+          className ?? ''
+        }`,
+      )}
```

Comment:
아래와 같이 `twMerge` 보다 `lib/utils` 의 `cn` 을 쓰는 것을 권장합니다.

```suggestion
      className={cn(
        'inline bg-violet-700 text-white hover:bg-violet-500 text-[16px] w-[140px]',
        className
      )}
```

Reply:
@f-lab-jay 아 깜빡했군요 twMerge에서 cn으로 수정했습니다.